StorageUtility/Windows/StorageUtility.cpp

```diff
@@ -203,6 +203,30 @@ eErrorCode GetStorageAdapterProperty( HANDLE Handle, sStorageAdapterProperty& Ad
     return( eErrorCode::None );
 }
 
+eErrorCode GetPhysicalDiskNumber(HANDLE Handle, U32& PhysicalDiskNumber)
```

Comment:
No, device path on windows is not unique for a Drive. Especially on the case I show you, when a drive on Rom mode has been performed DownloadAndExecute, the device path was changed then we lost to distinguish which our current drive are working